bitbots_hcm/src/bitbots_hcm/hardware_control_manager.py

```diff
@@ -172,6 +173,10 @@ def animation_callback(self, msg):
             out_msg.accelerations = [-1] * len(out_msg.joint_names)
             out_msg.velocities = [-1] * len(out_msg.joint_names)
             out_msg.max_currents = [-1] * len(out_msg.joint_names)
+            send_torque = False
+            if not len(msg.position.points[0].effort) == 0:
```

Comment:
as the codefactor.io says this is not optimal. maybe 
"if not msg.position.points[0].effort:"
works directly